packages/usa-button-group/src/styles/_usa-button-group.scss

```diff
@@ -18,6 +18,28 @@
     align-items: stretch;
     flex-direction: row;
   }
+
+  // Styles for nested button groups
+  .usa-button-group {
+    height: 100%;
+
+    .usa-button-group__item {
+      display: flex;
```

Comment:
Is this property required? Not seeing any effect on test when disabled.

Reply:
Good catch. It looks like this isn't needed anymore. I removed this in [8a926e3](https://github.com/uswds/uswds/pull/5885/commits/8a926e3ec27c9853ae66ff93eaa6d5541dc4fda7)